cockroachcloud/export-logs.md

```diff
@@ -129,21 +129,80 @@ Perform the following steps to enable log export from your {{ site.data.products
+1. Use one of the following Cloud API commands to enable log export for your {{ site.data.products.dedicated }} cluster. The first presents a basic configuration, where all logs are sent to AWS CloudWatch using the default settings. The second allows for more detailed customization of the logging configuration, such as the ability to send certain log channels to specific target log groups, or the ability to redact sensitive log entries.
 
-    {% include_cached copy-clipboard.html %}
-    ~~~shell
-    curl --request POST \
-      --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/logexport \
-      --header "Authorization: Bearer {secret_key}" \
-      --data '{"type": "AWS_CLOUDWATCH", "log_name": "{log_group_name}", "auth_principal": "{role_arn}"}'
-    ~~~
+    1. To enable log export for your {{ site.data.products.dedicated }} cluster with default logging configuration, issue the following Cloud API command:
 
-    Where:
-    - `{cluster_id}` is your {{ site.data.products.dedicated }} cluster ID as determined in step 3.
-    - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
-    - `{log_group_name}` is the target AWS CloudWatch log group you created in step 1.
-    - `{role_arn}` is the ARN for the `CockroachCloudLogExportRole` role you copied in step 8.
+        {% include_cached copy-clipboard.html %}
+        ~~~shell
+        curl --request POST \
+          --url https://cockroachlabs.cloud/api/v1/clusters/{cluster_id}/logexport \
+          --header "Authorization: Bearer {secret_key}" \
+          --data '{"type": "AWS_CLOUDWATCH", "log_name": "{log_group_name}", "auth_principal": "{role_arn}"}'
+        ~~~
+
+        Where:
+        - `{cluster_id}` is your {{ site.data.products.dedicated }} cluster ID as determined in step 3.
+        - `{secret_key}` is your {{ site.data.products.dedicated }} API key. See [API Access](console-access-management.html) for instructions on generating this key.
+        - `{log_group_name}` is the target AWS CloudWatch log group you created in step 1.
+        - `{role_arn}` is the ARN for the `CockroachCloudLogExportRole` role you copied in step 8.
+
+    1. To enable log export for your {{ site.data.products.dedicated }} cluster with custom logging configuration:
+
+        1. Consult the log export entry on the [Cockroach Cloud API Reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters/-cluster_id-/logexport) and select the **Schema** tab to view the supported log configuration options, and determine the customized logging configuration you would like to use.
+
+            For example, consider the following configuration:
+
+            {% include_cached copy-clipboard.html %}
+            ~~~json
+            {
+             type: AWS_CLOUDWATCH,
```

Comment:
I generally show configs in YAML, since it's more readable than JSON, but that's just me

Reply:
Cool, I'll try it next time.